NEWS.md

```diff
@@ -1,46 +1,63 @@
-# RevBayes 1.2.5 (unreleased)
+# RevBayes 1.2.5 (Dec 19, 2025)
+
+## Backwards-incompatible changes
+  * Remove `underPrior` argument to `mcmc.run( )` and `mcmcmc.run( )`.  You can use `model.ignoreAllData()` instead.
+  * Remove `dnLogexponential`. You can use `dnExponential(l) |> tnLog()` instead.
+  * Rename `tree.makeBifurcating()` to `tree.suppressOutdegreeOneNodes()`.
```

Comment:
I would phrase it as "Remove `tree.makeBifurcating()` in favor of `tree.suppressOutdegreeOneNodes()`". The method wasn't just renamed; its behavior was changed, too.

Reply:
Ah, I see.